demoscene/utils/groklinks.py

```diff
@@ -1710,6 +1710,18 @@ def __str__(self):
         return u"https://diskmag.conspiracy.hu/magazine/%s/#/edition=%s" % (magazine, edition)
 
 
+class EventsRetrosceneRelease(AbstractBaseUrl):
+    site =  Site(
+        "Events", long_name="Retroscene events", 
```

Comment:
Also, "Events" isn't really descriptive enough as link text - I'd suggest using "Retroscene events" as the standard label, not just the long name.
```suggestion
        "Retroscene events",
```